linux/image/Dockerfile-arm64

```diff
@@ -0,0 +1,3 @@
+FROM --platform=linux/arm64 phusion/holy-build-box:3.0
```

Comment:
I'm not sure this will work as Holy Build Box hasn't released an ARM version yet. 🤔

Reply:
They've provided one now added here https://github.com/phusion/holy-build-box/commit/578aa598888e63f9c6f9c15f393139a74f0ea342

https://hub.docker.com/r/phusion/holy-build-box/tags

<img width="1293" alt="Screenshot 2023-04-19 at 14 19 43" src="https://user-images.githubusercontent.com/19932401/233087336-a05a81ba-bd45-4a83-b839-965d6cc0615e.png">